### DISCUSSION TITLE
How to use slash commands

### DISCUSSION THREAD

**Discussion by Esteban-DA-COSTA** [discord-jda/JDA#1644] [Questions and Help] [answered]
I don't know how to create and use slash commands with 
JPA. Someone can help me ?

**Comment by MinnDevelopment** [comment, marked as answer]
#1501

**Reply by Esteban-DA-COSTA** [reply]
I checked it but it is available with 4.2.1_266?

**Reply by MinnDevelopment** [reply]
No, the PR is not merged yet. Read the description.
